### DISCUSSION TITLE
Question with chi parameter

### DISCUSSION THREAD

**Discussion by Nyanaaa** [pyscal/pyscal#130] [General]
Hi it's me again,

I was looking at your definition of the chi parameter and it seems that it is not implemented as it is in the paper you use as a reference.

Indeed, in the paper, they separate the angle distribution in 8 sections while you do it in 9 sections which make that it is not really usable if we want to apply the same method as the paper.

Is there a way to use the chi parameter in the same way as the paper by Ackland and Jones?

Sincerely yours

**Comment by srmnitc** [comment]
Hi, yes, pyscal provides an extra value just so that the entire range is covered. The range between cosines -0.705 to -0.195 is not used in the paper. So you can ignore the extra value and the rest should correspond to the paper values. For example for perfect HCP pyscal gives [3, 0, 6, 0, 21, 12, 0, 24, 0], whereas according to paper the value will be [3, 0, 6, 21, 12, 0, 24, 0]. You can see that pyscal has an extra zero and you can ignore it. In the paper they didnt include this range as it is zero for all cases they looked at. I saw that sometimes due to thermal vibrations there are angles that appear in the range and decided to keep it for completeness.
